### ISSUE TITLE
Test scripts missing?

### ISSUE THREAD

**Issue by dpage** [postgresml/postgresml#430]
`docker-compose up` results in:

```
...
postgres_1   | psql:tests/test.sql:17: error:  target | prediction 
postgres_1   | --------+------------
postgres_1   |       0 |          0
postgres_1   |       1 |          1
postgres_1   |       2 |          2
postgres_1   |       3 |          3
postgres_1   |       4 |          4
postgres_1   |       5 |          5
postgres_1   |       6 |          6
postgres_1   |       7 |          7
postgres_1   |       8 |          8
postgres_1   |       9 |          9
postgres_1   | (10 rows)
postgres_1   | 
postgres_1   | Time: 2.516 ms
postgres_1   | tests/joint_regression.sql: No such file or directory
postgresml_postgres_1 exited with code 3
```

**Comment by montanalow** [comment]
Reopening until we get these example scripts in the v2 CI.

**Comment by montanalow** [comment]
Fixed by #436
